Makefile

```diff
@@ -64,18 +65,19 @@ build: vendor/github.com/libgit2/git2go/v31/static-build ## Build heckler, usual
 	go build -o . -ldflags '$(GO_LDFLAGS)' ./...
 
 .PHONY: vet
-vet: ## Vet heckler, usually called inside the container
+vet: vendor/github.com/libgit2/git2go/v31/static-build ## Vet heckler, usually called inside the container
```

Comment:
Should this change and the change to the deb target be part of a separate commit, since they are not related to go.mod 1.16?